pom.xml

```diff
@@ -651,14 +651,7 @@ under the License.
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-site-plugin</artifactId>
-        <version>3.2</version>
-        <dependencies>
-          <dependency>
-            <groupId>org.apache.maven.doxia</groupId>
-            <artifactId>doxia-module-markdown</artifactId>
-            <version>1.3</version>
-          </dependency>
-        </dependencies>
+        <version>3.9.1</version>
```

Comment:
There is a reason why we keep the doxia plugin at 3.2. I've attempted to upgrade the doxia plugin to the latest a while ago, and although you see the mvn site command pass, it will end up generating a broken website.

Reply:
`doxia-module-markdown` is not a direct dependency in maven-site-plugin 3.2, but it is a dependency in 3.9.1.
https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-site-plugin/3.9.1

So I think it is fine to remove this dependency. And as I tested it, website htmls are fine. Maybe I have't checked the broken website htmls you talk about. If you could tell me what htmls are broken, I could test to verify.